registration/provision.py

```diff
@@ -65,11 +83,15 @@ def _provision_instance(sender, **kwargs):
     if application.instance is not None:
         logger.info('Email confirmed for user %s, but instance already provisioned.', user.username)
         return
+
+    design_fields_yaml = get_design_fields_as_yaml(application)
+
     with transaction.atomic():
         application.instance = production_instance_factory(
             sub_domain=application.subdomain,
             name=application.instance_name,
             email=application.public_contact_email,
+            configuration_extra_settings=design_fields_yaml,
```

Comment:
As noted in another comment, these settings should not be stored in `configuration_extra_settings`. Also, I don't think they need to be set at this stage at all - they can be set when spawning a new app server (on the new appserver itself).